x-pack/plugins/enterprise_search/public/applications/app_search/components/document_creation/creation_mode_components/paste_json_text.test.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { setMockValues, setMockActions } from '../../../../__mocks__/kea.mock';
+import { rerender } from '../../../../__mocks__';
+
+import React from 'react';
+import { shallow } from 'enzyme';
+import { EuiTextArea, EuiButtonEmpty, EuiButton } from '@elastic/eui';
+
+import { PasteJsonText, ModalHeader, ModalBody, ModalFooter } from './paste_json_text';
+
+describe('PasteJsonText', () => {
+  const values = {
+    textInput: 'hello world',
+    configuredLimits: {
+      engine: {
+        maxDocumentByteSize: 102400,
```

Comment:
You mock this value, and then don't actually test whether or not it's set. Did you mean to do so? Or are you mocking it here because it's required for render?

Reply:
It's required for render yeah. Usually for essentially text/copy only stuff I only bother examining it in tests if it has conditional logic attached, if it's just basic "I'm printing this value" I assume it will get caught in E2E testing or snapshot testing (although that leads me to wonder if we should be doing snapshot testing)

Reply:
Yeah I don't think it would get caught by anything, but I agree, I don't know that it's super important to test something like that explicitly in a unit test.